Addition of full file paths in security documentation (#8113)
opensearch-project/documentation-website: _install-and-configure/configuring-opensearch/security-settings.md

The Security plugin provides a number of YAML configuration files that are used to store the necessary settings that define the way the Security plugin manages users, roles, and activity within the cluster. For a full list of the Security plugin configuration files, see [Modifying the YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/).

> The following sections describe security-related settings in `opensearch.yml`, typically found at `/path/to/opensearch-{{site.opensearch_version}}/config/opensearch.yml`. To learn more about static and dynamic settings, see [Configuring OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index/).

Reviewer: Let's make all paths relative similar to @cwperks suggestions.

```suggestion
The following sections describe security-related settings in `opensearch.yml`. You can find the `opensearch.yml` in the `<OPENSEARCH_HOME>/config/opensearch.yml`. To learn more about static and dynamic settings, see [Configuring OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index/).
```
